scripts/actions/reset_default_app

```diff
@@ -0,0 +1,128 @@
+#!/bin/bash
+
+#=================================================
+# GENERIC STARTING
+#=================================================
+# IMPORT GENERIC HELPERS
+#=================================================
+
+source scripts/_common.sh
+source /usr/share/yunohost/helpers
+source scripts/_ynh_add_fpm_config
+
+#=================================================
+# MANAGE SCRIPT FAILURE
+#=================================================
+
+# Exit if an error occurs during the execution of the script
+ynh_abort_if_errors
+
+#=================================================
+# RETRIEVE ARGUMENTS
+#=================================================
+
+app=${YNH_APP_INSTANCE_NAME:-$YNH_APP_ID}
```

Comment:
```suggestion
app=$YNH_APP_INSTANCE_NAME
```

In case of multi instance app, if there are any bug in the core this line could reset the wrong app, so if `$YNH_APP_INSTANCE_NAME` is not defined, I'd prefer a fail here. (Same on other actions)

Reply:
If you know where in the core this behavior can be checked, I think that's an old error that has been fixed since.
It was an error at the beginning of actions.

Reply:
For actions : https://github.com/YunoHost/yunohost/blob/08f9091257feb7423ffb9e06f72845dcd4533131/src/yunohost/app.py#L1410
For config : https://github.com/YunoHost/yunohost/blob/08f9091257feb7423ffb9e06f72845dcd4533131/src/yunohost/app.py#L1473 and https://github.com/YunoHost/yunohost/blob/08f9091257feb7423ffb9e06f72845dcd4533131/src/yunohost/app.py#L1546

Reply:
Ok